pinax/stripe/models.py

```diff
@@ -298,8 +342,118 @@ class Charge(StripeObject):
+    display_name = models.TextField(blank=True, null=True)
+    email = models.TextField(blank=True, null=True)
+
+    legal_entity_address_city = models.TextField(null=True, blank=True)
+    legal_entity_address_country = models.TextField(null=True, blank=True)
+    legal_entity_address_line1 = models.TextField(null=True, blank=True)
+    legal_entity_address_line2 = models.TextField(null=True, blank=True)
+    legal_entity_address_postal_code = models.TextField(null=True, blank=True)
+    legal_entity_address_state = models.TextField(null=True, blank=True)
+    legal_entity_dob = models.DateField(null=True)
+    legal_entity_first_name = models.TextField(null=True, blank=True)
+    legal_entity_gender = models.TextField(null=True, blank=True)
+    legal_entity_last_name = models.TextField(null=True, blank=True)
+    legal_entity_maiden_name = models.TextField(null=True, blank=True)
+    legal_entity_personal_id_number_provided = models.BooleanField(default=False)
+    legal_entity_phone_number = models.TextField(null=True, blank=True)
+    legal_entity_ssn_last_4_provided = models.BooleanField(default=False)
+    legal_entity_type = models.TextField(null=True, blank=True)
+    legal_entity_verification_details = models.TextField(null=True, blank=True)
+    legal_entity_verification_details_code = models.TextField(null=True, blank=True)
+    legal_entity_verification_document = models.TextField(null=True, blank=True)
+    legal_entity_verification_status = models.TextField(null=True, blank=True)
+
+    managed = models.NullBooleanField(null=True)
+    metadata = JSONField(null=True)
+
+    product_description = models.TextField(null=True, blank=True)
+    statement_descriptor = models.TextField(null=True, blank=True)
+    support_email = models.TextField(null=True, blank=True)
+    support_phone = models.TextField(null=True, blank=True)
+
+    timezone = models.TextField(null=True, blank=True)
+
+    tos_acceptance_date = models.DateField(null=True)
+    tos_acceptance_ip = models.TextField(null=True, blank=True)
+    tos_acceptance_user_agent = models.TextField(null=True, blank=True)
+
+    transfer_schedule_delay_days = models.PositiveSmallIntegerField(null=True)
+    transfer_schedule_interval = models.TextField(null=True, blank=True)
+
+    transfer_schedule_monthly_anchor = models.PositiveSmallIntegerField(null=True)
+    transfer_schedule_weekly_anchor = models.TextField(null=True, blank=True)
+
+    transfer_statement_descriptor = models.TextField(null=True, blank=True)
+    transfers_enabled = models.BooleanField(default=False)
+
+    verification_disabled_reason = models.TextField(null=True, blank=True)
+    verification_due_by = models.DateTimeField(null=True, blank=True)
+    verification_timestamp = models.DateTimeField(null=True, blank=True)
+    verification_fields_needed = JSONField(null=True)
+
+    @property
+    def stripe_account(self):
+        return stripe.Account.retrieve(self.stripe_id)
```

Comment:
This should be a cached property, similar to 79424ed probably?!

Reply:
@blueyed - that hash you reference doesn't seem to link to a relevant change.

I'm going to leave these as properties for now, as this is what is used in general for accessing the Stripe object related to local instances. If we suddenly cache this, we could break people's code in subtle ways. They may, for example, call it multiple times on an instance and expect to see changes reflected in the returned value.